rust_src/Cargo.toml.in

```diff
@@ -63,6 +63,8 @@ window-system-x11 = []
 window-system-nextstep = []
 # Use the w32 window system
 window-system-w32 = []
+# Compiling for windows
```

Comment:
As @agraven points out, there is no need for this. Rust already defines `cfg(windows)`.